### PR TITLE
Ci/lighthouse workflow

### PR DESCRIPTION
**Pull request by nperon**
This PR adds a new workflow which consists of performance tests. Output is a set of indicators displayed in the PR page. 

The "action page" link redirects to the action page where we find a "lighthouse-result" link to the lighthouse report zip file. This file, once unzipped, is the lighthouse report in the shape of a static site. The latter static site can be displayed locally in your navigator.

In this project, I haven't added any limit in file ./github/lightouse-budget.json. We can discuss that issue before the PR is merged.

Note that the failure of lint-markdown is independent of this PR since no markdown file is submitted.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -122,3 +122,86 @@ jobs:
...
+              .map(formatRow)
+              .join('\n');
+            const content = linkToActionPage + '\n\n' + table;
+            core.setOutput("comment", content);
+
+      - name: Add Lighthouse stats as comment
+        id: comment_to_pr
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (amimart) [inline .github/workflows/lint.yml:204]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.OKP4_TOKEN }}
```

Can we use [bot-anik](https://github.com/bot-anik)'s token instead for commenting the PR ?

**Review by amimart — CHANGES_REQUESTED** [review]
A just have a single remark

File: .github/workflows/lint.yml (hunk 1 above)
Comment (nperon) [reply .github/workflows/lint.yml]: Correction for OKP4_TOKEN is now made everywhere.

**Comment by fredericvilcot** [comment]
@nperon I think we could put this PR as draft as it's not ready to review yet.

**Comment by ccamel** [comment]
Outdated. The functionality will be reintroduced using the recipe implemented in https://github.com/okp4/dataverse-portal/pull/57.

**Pull request closed without merging**